spec/system/prior_authority/primary_quote_summary_spec.rb

```diff
@@ -1,13 +1,14 @@
 require 'system_helper'
 
-RSpec.describe 'Prior authority applications - add service costs' do
+RSpec.describe 'Prior authority applications - add service costs', :javascript, type: :system do
```

Comment:
I'm really not a fan of making every system spec a JS spec. It slows the test suite down a lot. Can we not use `attach_file('file-field-selector', File.join(Rails.root, '/spec/fixtures/blah'))` to use the basic field attachment field that should be there when JS isn't available?

Reply:
The issue isn't the file upload, the issue is the suggestion box for the services which uses JS - it starts off as a completely different element which is a select element and is then converted to the searchable input field after the js runs - js needs to be enabled on every test which has going through the primary quote form in the before block (once we have the offences list in we will need to do this for that form too)

Reply:
@ivanELEC ah but we had that solved - because without JS that field is just a select, which is why the step helper selected a value from the select. Can't we just keep doing that so that we don't slow down the test suite?